static/docs/command-reference/remote/add.md

```diff
@@ -24,19 +24,19 @@ positional arguments:
 ## Description
 
 `name` and `url` are required. `url` specifies a location to store your data. It
-can be an SSH, S3 path, Azure, Google Cloud address, Aliyun OSS, local
-directory, etc. (See all the supported remote storage types in the examples
-below.) If `url` is a local relative path, it will be resolved relative to the
-current working directory but saved **relative to the config file location**
-(see LOCAL example below). Whenever possible DVC will create a remote directory
-if it doesn't exists yet. It won't create an S3 bucket though and will rely on
-default access settings.
+can be an SSH, S3 path, Azure, Google Drive path, Google Cloud address, Aliyun
```

Comment:
How is path different from address? Don't see that mentioned anywhere else 🤔

Reply:
good question, not related to PR though and a minor one ... @MaxRis feel free to change it path in all cases

Reply:
Have changed `Google Cloud address` to `Google Cloud path`. As @efiop mentioned, `address` is not used anywhere else within that context.

Reply:
Oh, I'm really sorry guys, I read it as "Google Drive path, Google Drive address". Thanks for fixing it!